lib/common/utils.c

```diff
@@ -120,9 +120,9 @@ check_number(const char *value)
 }
 
 gboolean
-check_positive_number(const char* value)
+check_positive_number(const char *value)
 {
-    if (safe_str_eq(value, CRM_INFINITY_S) || (crm_int_helper(value, NULL))) {
+    if (safe_str_eq(value, CRM_INFINITY_S) || (crm_parse_ll(value, NULL) > 0)) {
         return TRUE;
     }
     return FALSE;
```

Comment:
This function could just be reduced down to `return safe_str_eq(...) || crm_parse_ll(...)`, depending on your feelings about how readable that is.

Reply:
Might as well